Sources/Error.swift

```diff
@@ -2,22 +2,26 @@ public enum Error: ErrorType, Equatable {
     case General
     case InvalidUrl(String)
     case InvalidResponseData(NSData?)
-    case Error(NSError?)
+    case ErrorNS(NSError?)
```

Comment:
🎈I wouldn't use two types of errors... I would try to convert `NSError` to an `ErrorType` enum... Otherwise we are going to make it to complex...